includes/gateways/stripe/class-give-stripe.php

```diff
@@ -59,23 +60,49 @@ public function __construct() {
 		 */
 		public function includes() {
 
-			// Load Stripe SDK.
-			$stripe_sdk_compatibility = give_get_option( 'stripe_sdk_incompatibility', 'composer' );
+			// Include files which are necessary to load in admin but not in context of `is_admin`.
+			$this->include_admin_files();
```

Comment:
@mehul0810 I did not notice `is_admin` condition  in `include_admin_files` function. Will these files load on frontend too?

Reply:
I've implemented the `is_admin` condition within the `include_admin_files` fn to some of the files which are not disturbing the unit tests.